content/applications/inventory_and_mrp/purchase/manage_deals/manage.rst

```diff
@@ -219,30 +207,39 @@ Batch billing
 
 Vendor bills can be processed and managed in batches in the *Accounting* app.
 
-To do this, go to :menuselection:`Accounting app --> Vendors --> Bills`. Then, click the
-:guilabel:`checkbox` at the top left of the page, beside the :guilabel:`Number` column, under the
-:guilabel:`Create` button. This selects all existing vendor bills with a :guilabel:`Posted` or
-:guilabel:`Draft` :guilabel:`Status`.
+Navigate to :menuselection:`Accounting app --> Vendors --> Bills`. Then, click the
+:guilabel:`checkbox` in the top-left corner, beside the :guilabel:`Number` column, under the
+:guilabel:`New` button.
+
+This selects all existing vendor bills with a :guilabel:`Posted` or :guilabel:`Draft`
+:guilabel:`Status`.
+
+Click the :guilabel:`Actions` button, with the :icon:`fa-cog` :guilabel:`(cog)` icon, to export,
+delete, send & print the bills, and more.
 
-From here, click the :guilabel:`Action` gear icon to export, delete, or send & print the bills;
-click the :guilabel:`Print` icon to print the invoices or bills; or click :guilabel:`Register
-Payment` to create and process payments for multiple vendor bills at once.
+Click the :guilabel:`Print` button, with the :icon:`fa-print` :guilabel:`(print)` icon, to print the
+invoices or bills.
 
-When :guilabel:`Register Payment` is selected, a pop-up appears. In this pop-up window, select the
-appropriate journal in the :guilabel:`Journal` field, choose a payment date in the
-:guilabel:`Payment Date` field, and choose a :guilabel:`Payment Method`. There is also the option to
-:guilabel:`Group Payments` on this pop-up, as well.
+Click :guilabel:`Register Payment` to create and process payments for multiple vendor bills at once.
+
+.. note::
+   Only payments with their :guilabel:`Status` listed as :guilabel:`Posted` can be billed in
+   batches. Payments in the :guilabel:`Draft` stage *must* be posted before they can be included in
+   a batch billing.
 
-When ready, click the :guilabel:`Create Payment` button, which creates a list of journal entries on
-a separate page. This list of journal entries are all tied to their appropriate vendor bills.
+Clicking :guilabel:`Register Payment` opens a :guilabel:`Register Payment` pop-up window. From the
+pop-up window, select the :guilabel:`Journal` this bill should post to, choose a :guilabel:`Payment
+Date` for the bill, and select a :guilabel:`Payment Method`.
+
+There is also the option to :guilabel:`Group Payments` together from this pop-up window, as well. If
+this checkbox is ticked, only one payment is created by partner (bank), instead of one per bill.
```

Comment:
I didn't see this option. Is there something I need to enable? O:
![image](https://github.com/odoo/documentation/assets/23611711/906e34e2-1851-4cc0-b50a-dabd53e1e990)